MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -963,9 +982,36 @@ private void BuildStreamVideoItem(
 
             var audioStreamWithSupportedCodec = candidateAudioStreams.Where(stream => ContainerHelper.ContainsContainer(audioCodecs, false, stream.Codec)).FirstOrDefault();
 
-            var directAudioStream = audioStreamWithSupportedCodec?.Channels is not null && audioStreamWithSupportedCodec.Channels.Value <= (playlistItem.TranscodingMaxAudioChannels ?? int.MaxValue) ? audioStreamWithSupportedCodec : null;
-
-            var channelsExceedsLimit = audioStreamWithSupportedCodec is not null && directAudioStream is null;
+            var channelsExceedsLimit = audioStreamWithSupportedCodec is not null && audioStreamWithSupportedCodec.Channels > (playlistItem.TranscodingMaxAudioChannels ?? int.MaxValue);
+
+            var directAudioStreamSatisfied = audioStreamWithSupportedCodec is not null && !channelsExceedsLimit &&
+                                             options.Profile.CodecProfiles
+                                                 .Where(i => i.Type == CodecType.VideoAudio &&
+                                                             i.ContainsAnyCodec(audioStreamWithSupportedCodec.Codec, container) &&
+                                                             i.ApplyConditions.All(applyCondition =>
```

Comment:
What happened to the indentation? Here and above. Are we going to align at the end of `=`, `=>`?

Reply:
This is just IDE auto indent. Where should it indent with?